source/gear/swordofthesun-lethal.js

```diff
@@ -0,0 +1,44 @@
+const { GearTemplate } = require('../classes');
+const { ESSENCE_MATCH_STAGGER_FOE } = require('../constants');
+const { getModifierCategory } = require('../modifiers/_modifierDictionary');
+const { dealDamage, changeStagger, combineModifierReceipts, removeModifier, generateModifierResultLines } = require('../util/combatantUtil');
+const { damageScalingGenerator } = require('./shared/scalings');
+
+module.exports = new GearTemplate("Lethal Sword of the Sun",
+	[
+		["use", "Removes all buffs from a foe, then deal <@{damage}> @{essence} damage to a them, plus 30 damage per unique buff removed"],
```

Comment:
```suggestion
		["use", "Removes all buffs from a foe, then deal <@{damage} + 30 per unique buff removed> @{essence} damage to a them"],
```
This applies to all three variants, but only commenting here to avoid duplication cluttering things up too much. In PotL we're starting a convention where triangle brackets `< >` group formulae, so the extra damage for buff removal would go in there.